support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -252,6 +255,22 @@ export function ThreeTierLanding(): JSX.Element {
 			? 'monthly'
 			: (productType.toLowerCase() as 'monthly' | 'annual');
 
+	const isCardUserSelected = (
+		cardPrice: number,
+		cardPriceDiscount?: number,
+	): boolean => {
+		const cardPriceToCompare = cardPriceDiscount ?? cardPrice;
+		const hasUrlSelectedAmount = !isNaN(Number(urlSelectedAmount));
+
+		if (!hasUrlSelectedAmount) {
+			return false;
+		}
+		return (
+			Object.prototype.hasOwnProperty.call(paymentFrequencyMap, productType) &&
```

Comment:
I've found the [`in` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) a clearer way to express this.

```suggestion
			productType in paymentFrequencyMap &&
```